draft-dns-zone-digest.xml

```diff
@@ -953,14 +955,18 @@ RR(i) = owner | type | class | TTL | RDATA length | RDATA
           only with DNSSEC validation.
         </t>
       </section>
-      <section title="Attacks Utilizing the Zone Digest">
+      <section title="Attacks Utilizing &RRNAME; Queries">
         <t>
           Nothing in this specification prevents clients from making,
-          and servers from responding to, &RRNAME; queries.  One might
-          consider how well &RRNAME; responses could be used in
-          a distributed denial-of-service amplification attack.
+          and servers from responding to, &RRNAME; queries.
+          Servers SHOULD NOT calculate zone digests dynamically (for
+          each query) as this can be used as a CPU resource exhaustion
+          attack.
```

Comment:
Yeah, this is worth adding -- I didn't really think implementations might do this, but I guess I *could* see someone:
A: doing this on AXFR
B: allowing AXFR from arbritary places.